tests/drivers/memc/stm32_sdram/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_ZTEST=y
+
+CONFIG_MEMC=y
+CONFIG_MEMC_STM32_SDRAM=y
```

Comment:
I would prefer we can get rid of this particular symbols activation and rely on device tree instead.

Reply:
removed (using Kconfig default according to DT)